scripts/mkcloud

```diff
@@ -1077,6 +1077,8 @@ Optional
         It's not deployed by default. Set to 1 to deploy murano.
     want_trove=0 (default=1)
         Deployed by default. Set to 0 to prevent trove from being deployed.
+    want_aodh=0 (default=0)
```

Comment:
let's keep it as 0 for the time being so I can only apply it for my jobs testing. Once all works, will change to 1